tests/test_deserialization.py

```diff
@@ -951,6 +951,7 @@ def test_structured_dict_key_value_deserialization(self):
     def test_url_field_deserialization(self):
         field = fields.Url()
         assert field.deserialize("https://duckduckgo.com") == "https://duckduckgo.com"
+        assert field.deserialize("file:///tmp/tmp1234") == "file:///tmp/tmp1234"
```

Comment:
Can you move the testing to `test_validate.py` and add a focused test for file URLs there?